package.json

```diff
@@ -30,11 +31,13 @@
   },
   "scripts": {
     "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
-    "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
+    "lint": "eslint lib/ bin/ test/ --cache --fix",
     "lint:md": "remark  -f '*.md' 'doc/*.md'",
     "lint:text": "textlint '*.md' 'doc/*.md'",
-    "test": "nyc --reporter=text mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
-    "test:coverage": "nyc --reporter=lcov mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "prettier": "prettier --config .prettierrc.json --write '**/**/**/**/*.js' '**/**/**/*.js' '**/**/*.js' '**/*.js' '*.js'",
+    "prettier:text": "prettier 'README.md' 'doc/*.md' 'doc/**/*.md' --no-config --tab-width 4 --print-width 120 --write --prose-wrap always",
```

Comment:
Are the new targets (prettier and prettier:text) included in the corresponding .md describing the different targets? Maybe yes, but it's a bit hard to find it in so large PR :) Could you help providing a pointer, please?

Reply:
Fixed ea6d399